Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -53,8 +53,9 @@ inline CGPoint __CreateCGPointWithTransform(CGFloat x, CGFloat y, const CGAffine
     bool figureClosed;
     CGPoint currentPoint{ 0, 0 };
     CGPoint startingPoint{ 0, 0 };
+    const CGAffineTransform* lastTransform;
```

Comment:
Any reason why this isnt a copy?

---
In reply to: [88586100](https://github.com/Microsoft/WinObjC/pull/1384#discussion_r88586100) [](ancestors = 88586100)